components/UI/navbar.tsx

```diff
@@ -336,11 +339,11 @@ const Navbar: FunctionComponent = () => {
               </div>
             </div>
             <div className="flex flex-col items-center my-4 w-full">
-              <div className="text-background">
+              <div className={styles.connectButtonContainer}>
                 <Button onClick={onTopButtonClick}>{topButtonText()}</Button>
               </div>
               <div className="flex">
-                <div className="rounded-full shadow-gray-400 p-3 modified-cursor-pointer hover:scale-105 ease-in duration-300 mt-2">
+                <div className="rounded-full shadow-gray-400 p-3 modified-cursor-pointer hover:scale-105 ease-in duration-300 mt-6">
```

Comment:
should we add this styling in a css class and add it here? seems to be repeated in two places. Would be easier to modify both at once next time from one place